app/src/backend/dependency.ts

```diff
@@ -150,6 +157,7 @@ export function installAnaconda(
   let destination = path.join(tmpdir, basename);
 
   let initSJCloudHome = () => {
+    startInitSJCloud = process.hrtime();
```

Comment:
I'd probably use [`performance.now`](https://nodejs.org/api/perf_hooks.html#perf_hooks_performance_now) over `process.hrtime` so you can work just with milliseconds rather than [seconds, milliseconds].